install.sh

```diff
@@ -112,7 +112,7 @@ function k() {
       echo -e "Current Context$UNCHANGED:\n$KUBE_CONTEXT"
     fi
     __easy_kubectl_export_variables $VARIABLES_FN
-  elif [[ $# -lt 2 ]]; then
+  elif [[ $# -eq 0 ]]; then
```

Comment:
There is a usage `k <keyword>` to select namespace directly if only one namespace is matched.
So you should revert this line. (If you want to pass only one argument to kubectl , you can directly use `kubectl <arg>`)

Reply:
recover